lib/generators/module/templates/README.md

```diff
@@ -0,0 +1,28 @@
+# Vsp
+Short description and motivation.
+
+## Usage
+How to use my plugin.
+
+## Installation
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'vsp'
+```
+
+And then execute:
+```bash
+$ bundle
+```
+
+Or install it yourself as:
+```bash
+$ gem install vsp
+```
+
+## Contributing
+Contribution directions go here.
```

Comment:
Do we have some standard text that can go here?
Since all these Engines live in the same `vets-api` codebase, should all of these Engine README sections be identical?  Or will they vary by team?